### DISCUSSION TITLE
rename ignition-* to gz-*

### DISCUSSION THREAD

**Discussion by PSandro** [acxz/gazebo-arch#63] [Ideas]
It would be nice to rename all the aur packages from `ignition-*` to `gz-*` 
Gazebo itself did this change for all ignition packages, see [blog post here](https://community.gazebosim.org/t/a-new-era-for-gazebo/1356)

> [...] In summary, each place where the name “Ignition” or “ign” is used will be converted to a Gazebo counterpart (“Gazebo” or “gz”). The Ignition logo will also be replaced by the Gazebo logo.

Especially for new users of gazebo, it might be confusing to still find the name ignition in the aur while on the gazebo website they talk about "gz" and "Gazebo".

**Comment by acxz** [comment]
This is a very good idea. I'll try to rename the packages to their `gz-*` names by the end of next week. Thanks for the suggestion!
